misc/setup.py

```diff
@@ -2,6 +2,8 @@
 
 print("-*-*-*-*-*-*-*-*-*-*-*Setup begins*-*-*-*-*-*-*-*-*-*-*-*-*-")
 
-delete_for_users()
+delete_for_users(gis_online, ignore_accounts_online, target_accounts_online)
+delete_for_users(gis_playground, ignore_accounts_playground, target_accounts_playground)
+publish_data(gis_online, data_paths)
```

Comment:
@Yongyao for this specific notebook, you need to publish data using the `arcgis_python` account and not the `api_data_owner`. So, instead of passing `gis_online` connection, you can pass the connection that pertains to the `arcgis_python` user. Also, pass only the specific file that meets this exception.

Reply:
@AtmaMani The csv is the only file I need to publish at this point. I am not publishing it under `api_data_owner`. I am publishing it under the test admin account and sharing it with everyone. Would that be okay?